Plogon/BuildProcessor.cs

```diff
@@ -520,6 +520,15 @@ public async Task<BuildResult> ProcessTask(BuildTask task, bool commit, string c
                         file.CopyTo(Path.Combine(repoOutputDir.FullName, file.Name), true);
                     }
 
+                    var imagesSourcePath = Path.Combine(dpOutput.FullName, "images");
```

Comment:
Would it be safer to copy out of the manifest folder? Harder to get that here, but I think it's rare for people to set up DP to copy images.

Reply:
Hmm I thought that's how everyone did it, but since the images are guaranteed to be in the manifest folder at least, I'll have a look at that.